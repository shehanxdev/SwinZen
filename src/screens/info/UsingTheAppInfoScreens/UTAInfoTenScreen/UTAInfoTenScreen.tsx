import React from 'react';
import { View } from 'react-native';

import { Text } from '@sz/components';
import { tw } from '@sz/config';
import { TextAlignment, TextVariant } from '@sz/constants';

import { BaseInfoScreen } from '../../components';

export function UTAInfoTenScreen() {
  return (
    <BaseInfoScreen>
      <View style={tw`px-4 py-6`} testID="UTAInfoTenScreenTestID">
        <Text variant={TextVariant.Body2Regular} textAlign={TextAlignment.Auto}>
          UTAInfoTenScreen
        </Text>
      </View>
    </BaseInfoScreen>
  );
}
