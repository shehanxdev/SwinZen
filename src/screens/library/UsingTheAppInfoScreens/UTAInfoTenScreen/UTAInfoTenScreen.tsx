import React from 'react';
import { View } from 'react-native';

import { Text } from '@sz/components';
import { tw } from '@sz/config';
import { Color, TextAlignment, TextVariant } from '@sz/constants';

import { BaseLibraryScreen } from '../../components';

export function UTAInfoTenScreen() {
  return (
    <BaseLibraryScreen>
      <View style={tw`mx-4 mt-8`} testID="UTAInfoTenScreenTestID">
        <Text color={Color.Neutral.White} variant={TextVariant.Body1Regular} textAlign={TextAlignment.Auto}>
          {`SwingZenâ€™s concept of AI Pro instruction is to provide a similar process of instruction that any PGA/LPGA Pro would utilize.
          \nThe first step in building a quality swing is to start with your setup. Setup is your swing foundation. Once you get that right you can build off of it. Your AI Pro swing results are listed in priority starting with setup and progressing through the swing. Before you focus on the next thing, master the first priority item and then move on to the next.`}
        </Text>
      </View>
    </BaseLibraryScreen>
  );
}
