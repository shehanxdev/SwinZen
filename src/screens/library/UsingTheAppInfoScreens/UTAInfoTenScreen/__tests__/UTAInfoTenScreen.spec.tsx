import { render } from '@testing-library/react-native';
import * as React from 'react';

import { UTAInfoTenScreen } from '../UTAInfoTenScreen';

describe('UTAInfoTen Screen', () => {
  const testID = 'UTAInfoTenScreenTestID';

  const getRenderedScreen = () => render(<UTAInfoTenScreen />);

  it(`should render UTAInfoTen Screen correctly`, () => {
    const renderer = getRenderedScreen();
    const renderTree = renderer.toJSON();
    expect(renderTree).toMatchSnapshot();
  });

  it('should find the UTAInfoTen Screen via testID', () => {
    const { getByTestId } = getRenderedScreen();
    const foundScreen = getByTestId(testID);
    expect(foundScreen).toBeTruthy();
  });
});
