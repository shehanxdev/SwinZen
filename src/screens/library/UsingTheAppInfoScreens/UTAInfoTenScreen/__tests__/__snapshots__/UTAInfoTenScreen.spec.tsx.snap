// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UTAInfoTen Screen should render UTAInfoTen Screen correctly 1`] = `
<BVLinearGradient
  colors={
    [
      4279917603,
      4278984207,
      4278650631,
    ]
  }
  endPoint={
    {
      "x": 0.5,
      "y": 1,
    }
  }
  locations={null}
  startPoint={
    {
      "x": 0.5,
      "y": 0,
    }
  }
  style={
    {
      "flexBasis": "0%",
      "flexGrow": 1,
      "flexShrink": 1,
      "position": "relative",
    }
  }
>
  <Image
    resizeMode="cover"
    source={
      {
        "testUri": "../../../src/assets/images/grassBackground.jpg",
      }
    }
    style={
      {
        "opacity": 0.1,
        "position": "absolute",
      }
    }
  />
  <RCTSafeAreaView
    style={
      {
        "height": "100%",
        "paddingTop": 50,
      }
    }
  >
    <RCTScrollView
      contentContainerStyle={
        {
          "flexGrow": 1,
        }
      }
      keyboardShouldPersistTaps="handled"
    >
      <View>
        <View
          style={
            {
              "marginLeft": 16,
              "marginRight": 16,
              "marginTop": 32,
            }
          }
          testID="UTAInfoTenScreenTestID"
        >
          <Text
            style={
              [
                {
                  "color": "#FFFFFF",
                  "lineHeight": 28,
                },
                {
                  "fontFamily": "SourceSansPro-Regular",
                  "fontSize": 18,
                  "fontWeight": "normal",
                  "textAlign": "auto",
                },
              ]
            }
          >
            SwingZenâ€™s concept of AI Pro instruction is to provide a similar process of instruction that any PGA/LPGA Pro would utilize.
          
The first step in building a quality swing is to start with your setup. Setup is your swing foundation. Once you get that right you can build off of it. Your AI Pro swing results are listed in priority starting with setup and progressing through the swing. Before you focus on the next thing, master the first priority item and then move on to the next.
          </Text>
        </View>
      </View>
    </RCTScrollView>
  </RCTSafeAreaView>
</BVLinearGradient>
`;
